packages/cli/src/utils/build.ts

```diff
@@ -1,15 +1,17 @@
 import { execa } from "execa";
 import { copyFileSync, mkdirSync, readdirSync, rmSync } from "fs";
 import path from "path";
+import { getOutDir } from "./forgeConfig";
 
-export async function forgeBuild(out = "out", options?: { clear?: boolean }) {
+export async function forgeBuild(options?: { clear?: boolean }) {
   if (options?.clear) {
+    const out = await getOutDir();
     console.log("Clearing forge build output directory", out);
     rmSync(out, { recursive: true, force: true });
   }
 
   console.log("Running forge build");
-  const child = execa("forge", ["build", "-o", out], {
```

Comment:
A change I wanna make soon is generate ABIs separately from the standard forge output with
```
forge build --out abi --extra-output-files abi
```
Then we can `.gitignore` any non `*.abi.json` file so that ABIs can be checked in (and CI jobs to ensure these are up to date).

Reply:
> We should check whether that's still the case - if forge's output has changed all good

Tested it on a couple projects, the output is flat, and skipping explicit out doesn't change it